crypto/bio/b_sock2.c

```diff
@@ -223,11 +259,8 @@ int BIO_listen(int sock, const BIO_ADDR *addr, int options)
     }
 # endif
 
-    if (bind(sock, BIO_ADDR_sockaddr(addr), BIO_ADDR_sockaddr_size(addr)) != 0) {
-        SYSerr(SYS_F_BIND, get_last_socket_error());
-        BIOerr(BIO_F_BIO_LISTEN, BIO_R_UNABLE_TO_BIND_SOCKET);
+    if (!BIO_localaddr(sock, addr, options))
```

Comment:
Why is it called localaddr and not bind? It looks like bind, it barks like bind...

Reply:
Renamed BIO_localaddr to BIO_bind.

(Which was irritatingly difficult to do).